public/app/features/dashboard-scene/scene/ShareQueryDataProvider.ts

```diff
@@ -120,6 +120,13 @@ export class ShareQueryDataProvider extends SceneObjectBase<ShareQueryDataProvid
     }
     return false;
   }
+
+  public getInterval() {
```

Comment:
If the macro just read interval from data.request would we even need this function?

Reply:
no need for that, tru 🙈